app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -536,20 +661,50 @@ class SamplesHeatmapView extends React.Component {
   });
 
   handleSelectedOptionsChange = newOptions => {
-    const excluding = ["dataScaleIdx", "sampleSortType", "taxaSortType"];
-    const shouldRefetchData = difference(keys(newOptions), excluding).length;
-    this.setState(
-      {
-        selectedOptions: assign(this.state.selectedOptions, newOptions),
-      },
-      shouldRefetchData ? this.updateHeatmap : null
-    );
+    if (this.props.allowedFeatures.includes("heatmap_filter_fe")) {
+      const frontend_filters = [
+        "categories",
+        "subcategories",
+        "thresholdFilters",
+        "readSpecificity",
+        "metric",
+      ];
+      const backend_filters = ["species", "background", "taxonsPerSample"];
```

Comment:
Use camelCase

Reply:
We can definitely do the taxonsPerSample on the client side (should rename `taxons` -> `taxa`)... Eventually we should do the `species` too.